www/src/components/repos/Docker.tsx

```diff
@@ -157,6 +184,17 @@ function ImageVersionPicker({ image }: any) {
             )}
           />
         ))}
+        {hasNextPage ? (
```

Comment:
Would `{hasNextPage && (` work here instead?

Reply:
Actually, this should be moved to `dropdownFooter` prop. See the example from Storybook:

```
dropdownFooter={
  shownLimit < items.length && (
    <ListBoxFooterPlus>View more</ListBoxFooterPlus>
  )
}
```

Reply:
Dropdown closing on click will also stop being a problem when this prop will be used.

Reply:
No... Because of problem number two

Reply:
It was my guess as it works like this here: https://pluralsh-design.web.app/?path=/story/select--default (see "Click me!" button).

Reply:
OH that's great!